ci-operator/config/openshift/image-customization-controller/openshift-image-customization-controller-main.yaml

```diff
@@ -55,11 +55,11 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-ipv4
-- as: e2e-metal-ipi-serial-ipv4
+- as: e2e-metal-ipi-serial-virtualmedia
```

Comment:
Side note: I think we also need a v6 job on ICC because of its networking features.